### PR TITLE
[ES] Clean up `Tournaments/MWC/2014`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/MWC/2014/es.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
-outdated_translation: true
+outdated_translation: false
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2014/es.md:2]: it's still outdated, because it is not up to date with the english version

**Review by Walavouchey — COMMENTED** [review]
https://github.com/ppy/osu-wiki/pull/9053 does a lot of bulk changes, one of them being that it uses "beatmaps" instead of "mapa", contrary to this pr. which should it be?

cc/ @agatemosu

**Comment by agatemosu** [comment]
> https://github.com/ppy/osu-wiki/pull/9053 does a lot of bulk changes, one of them being that it uses "beatmaps" instead of "mapa", contrary to this pr. which should it be?
> 
> cc/ @agatemosu 

I made the changes because "beatmap" does not have a translation into Spanish and the wiki sometimes uses "beatmap" and other times "map". I don't know if in Spanish I should only put "map".

**Review by TicClick — APPROVED** [review]
merging (no native review for one week or longer). no `no_native_review: true` since the translation is outdated anyway and this PR is a small cleanup

**Pull request merged**